compose/django/pyproject.toml

```diff
@@ -1,15 +1,30 @@
-[[source]]
-url = "https://pypi.org/simple"
-verify_ssl = true
-name = "pypi"
+[tool.poetry]
+name = "transmission"
+version = "0.8.0"
+description = "A Django REST API built to proxy Ethereum requests via ShipChain Engine."
+
+license = "Apache-2.0"
+
+authors = [
+    "Adam Hodges <ahodges@shipchain.io>"
+]
+
+readme = 'README.md'  # Markdown files are supported
+
+repository = "https://github.com/ShipChain/transmission"
+homepage = "https://transmission.shipchain.io"
+
+keywords = ['transmission', 'shipchain', 'logistics']
+
+[tool.poetry.dependencies]
+python = "~3.6.5"
```

Comment:
Wouldn't it be preferable to lock the python version? `3.6.5` instead of `~3.6.5`